packages/amplify-environment-parameters/src/environment-parameter-manager.ts

```diff
@@ -78,7 +86,7 @@ class EnvironmentParameterManager implements IEnvironmentParameterManager {
     return !!this.resourceParamManagers[getResourceKey(category, resource)];
   }
 
-  save(): void {
+  async save(): Promise<void> {
```

Comment:
no await inside the body?

Reply:
it can be synchronous for now, but we are getting ready to add some async stuff